packages/solarwinds-apm/src/sampling/http.ts

```diff
@@ -109,62 +107,63 @@ export class HttpSampler extends Sampler {
 
   /** Settings update loop */
   #loop() {
-    context
-      .with(suppressTracing(context.active()), async () => {
```

Comment:
What did this do before?

Reply:
This is to not create spans for the HTTP call to the sampler, I had to move it around for it to work properly with context propagation in the browser.